data/language/dutch.txt

```diff
@@ -53,9 +53,9 @@ STR_0051    :Attractie
 STR_0052    :Attractie
 STR_0053    :Hyper-Twisterachtbaan
 STR_0054    :Houten achtbaan
-STR_0055    :Zijfrictieachtbaan
+STR_0055    :Zijfrictie-achtbaan
 STR_0056    :Wilde muis
-STR_0057    :Multidimensieachtbaan
+STR_0057    :Multidimensie-achtbaan
```

Comment:
De verandering 'miniachtbaan -> mini-achtbaan' is overigens eerder wel doorgevoerd. Mogelijk maakt een koppelteken het leesbaarder, zeker voor jongere spelers. Beide varianten zijn overigens correct.